packages/xdl/src/credentials/Credentials.ts

```diff
@@ -1,42 +1,40 @@
-/* @flow */
-
 import Api from '../Api';
 import * as Ios from './IosCredentials';
 
 export type Credentials = Ios.Credentials; // can't import android types from typescript
 
 export type CredentialMetadata = {
-  username: string,
-  experienceName: string,
-  bundleIdentifier?: string,
-  platform: string,
+  username: string;
+  experienceName: string;
+  bundleIdentifier?: string;
+  platform: string;
 };
 
 export { Ios };
 
 export async function credentialsExistForPlatformAsync(
   metadata: CredentialMetadata
-): Promise<?Credentials> {
-  const creds = await fetchCredentials(metadata, false);
-  return !!(creds: any); // !! performed on awaited creds
+): Promise<boolean> {
+  const credentials = await fetchCredentials(metadata, false);
+  return !!credentials;
 }
 
 export async function getEncryptedCredentialsForPlatformAsync(
   metadata: CredentialMetadata
-): Promise<?Credentials> {
+): Promise<Credentials | null> {
```

Comment:
I think it's never `null`, it's either `Credentials` or `undefined`. Same for the two other functions below.

https://github.com/expo/universe/blob/master/server/www/src/api/credentials.js#L69

Reply:
Ah, good call!